control-plane/tests/test_service_persistence_restore_readiness.py

```
@@ -54,6 +54,22 @@ def __getattr__(self, name: str) -> object:
 
 
 class RestoreReadinessPersistenceTests(ServicePersistenceTestBase):
+    def test_service_wires_restore_readiness_internal_collaborators(self) -> None:
+        service = AegisOpsControlPlaneService(
```

Ruff (0.15.10)

[error] Undefined name `AegisOpsControlPlaneService`

(F821)